### PR TITLE
Migrate bicep example: 101/webapp-privateendpoint-vnet-injection

### PR DESCRIPTION
**Pull request by StephenWeatherford**
Added bicep support to this sample by integrating bicep.main from the example in https://github.com/Azure/bicep/tree/main/docs/examples/101/webapp-privateendpoint-vnet-injection

The corresponding PR for the modified bicep example is here: https://github.com/Azure/bicep/pull/3292

### CODE REVIEW

File: quickstarts/microsoft.web/webapp-privateendpoint-vnet-injection/main.bicep [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+])
+param skuFamily string = 'P1v2'
+
+@description('Name of your Private Endpoint')
+param privateEndpointName string = 'PrivateEndpoint1'
+
+@description('Link name between your Private Endpoint and your Web App')
+param privateLinkConnectionName string = 'PrivateEndpointLink1'
+
+var webapp_dns_name = '.azurewebsites.net'
```
Comment (bmoore-msft) [inline quickstarts/microsoft.web/webapp-privateendpoint-vnet-injection/main.bicep:61]: use the environment() function to get the endpoint

File: quickstarts/microsoft.web/webapp-privateendpoint-vnet-injection/main.bicep [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+var privateDNSZoneName = 'privatelink.azurewebsites.net'
```
Comment (bmoore-msft) [inline quickstarts/microsoft.web/webapp-privateendpoint-vnet-injection/main.bicep:62]: use the environment() function to get the endpoint

File: quickstarts/microsoft.web/webapp-privateendpoint-vnet-injection/main.bicep [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+      {
+        name: 'delegation'
+        properties: {
+          serviceName: 'Microsoft.Web/serverfarms'
+        }
+      }
+    ]
+    privateEndpointNetworkPolicies: 'Enabled'
+  }
+  dependsOn: [
```
Comment (bmoore-msft) [inline quickstarts/microsoft.web/webapp-privateendpoint-vnet-injection/main.bicep:101]: See: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties for the proper sort order of properties

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: quickstarts/microsoft.web/webapp-privateendpoint-vnet-injection/main.bicep (hunk 1 above)
Comment (StephenWeatherford) [reply quickstarts/microsoft.web/webapp-privateendpoint-vnet-injection/main.bicep:61]: How?  I don't see .azurewebsites: https://docs.microsoft.com/en-us/azure/azure-resource-manager/templates/template-functions-deployment#example-1

**Pull request merged**